### ISSUE TITLE
Applicant - Indicate approx word limit for skills 

### ISSUE THREAD

**Issue by JasmitaChawla** [GCTC-NTGC/TalentCloud#556]
# Description
Indicate an approximate word limit for _How I Acquired This Skill_, so users have a rough idea of how much they are expected to write. 

Note: The text has to be reviewed. 

# Images
<img width="522" alt="indicate approx word limit" src="https://user-images.githubusercontent.com/38079220/50353253-a27c0680-0515-11e9-9d60-2d22903a70bf.png">

**Comment by gobyrne** [comment]
I agree we should add something that helps applicants meet our expectations.

@laurenoncoffee can you please comment on whether or not you like word count for this. Also how many words you would recommend.

**Comment by laurenoncoffee** [comment]
i think my preference would be a pop-up with explanation, and maybe links to a few examples. i'm nervous about a word limit... but maybe we could say "suggested length: # sentences/words". We're noticing a lot of responses that are too short, which is a bigger problem for us in real world conditions, as its happening a lot. Managers report giving people benefit of the doubt, but they're not sure if people have the skills a lot of the time based on the limited response

**Comment by gobyrne** [comment]
Yeah, like in the mock-up above. I think it's more of an approx word count. I don't think we need to put a hard cap on length.
